packages/braid-design-system/src/lib/components/Card/Card.tsx

```diff
@@ -68,16 +70,19 @@ export const Card = ({
     ]);
   }
 
+  const roundingForTheme = !isLegacyTheme ? borderRadius : resolvedRounding;
```

Comment:
I would flip the condition to make the exception more obvious i.e.

> If (exception) do something
> Else do the default

It's also consistent with the other ternaries introduced in this PR.